v20.2/alter-table.md

```diff
@@ -19,9 +19,7 @@ Subcommand | Description | Can combine with other subcommands?
 [`ADD COLUMN`](add-column.html) | Add columns to tables. | Yes
 [`ADD CONSTRAINT`](add-constraint.html) | Add constraints to columns. | Yes
 [`ALTER COLUMN`](alter-column.html) | Change or drop a column's [`DEFAULT` constraint](default-value.html) or [`NOT NULL` constraint](not-null.html). | Yes
```

Comment:
This is sort of out dated now. The description only covers some cases. Maybe we should actually have the description be more vague and have them click through to the ALTER COLUMN page.